aimmo-game/service.py

```diff
@@ -35,13 +35,13 @@ def to_cell_type(cell):
     return 0
 
 
-def player_dict(avatar):
+def player_dict(avatar, min_x, min_y):
     # TODO: implement better colour functionality: will eventually fall off end of numbers
     colour = "#%06x" % (avatar.player_id * 4999)
     return {
         'id': avatar.player_id,
-        'x': avatar.location.x,
-        'y': avatar.location.y,
+        'x': avatar.location.x-min_x,
```

Comment:
This sounds like it would be a significant change from the current player-facing api.

If you can't rely on the coordinates of avatars or bits of the map remaining constant from one call to the next it's going to make persisting world data in a fog situation really confusing.

(We've been imagining that if you're writing an avatar for a simulation with fog, you can persist the data to your local avatar each time, to gradually build up a picture of the whole world. If these coordinates can change it's going to make that confusing for students.)